test/JDBC/expected/BABEL-3655-vu-cleanup.out

```diff
@@ -0,0 +1,6 @@
+DROP DATABASE babel_3655_test_db
+GO
+~~ERROR (Code: 911)~~
+
+~~ERROR (Message: database "babel_3655_test_db" does not exist)~~
```

Comment:
We don't need a cleanup file?

Reply:
DROP DATABASE is the core of the test, so we decided to include it in the verify file and have additional DROP command in the cleanup file for additional check. Do you think it is redundant?

Reply:
Cleanup file is optional. We can remove this file